api-tests/checkout-tests/checkout-for-ecommerce-api.tests.json

```diff
@@ -340,9 +340,18 @@
 							"        return rest;",
 							"    });",
 							"    const paymentTokens = response.payments.map(e => e.paymentToken);",
-							"",
+							"    ",
 							"    pm.expect(response.status).eql(expectedWithoutAuthTokenAndTransactionId.status);",
-							"    pm.expect(paymentsWithoutPaymentTokenAndTransferList).eql(expectedWithoutAuthTokenAndTransactionId.payments);",
+							"    let i = 0;",
+							"        pm.expect(paymentsWithoutPaymentTokenAndTransferList).to.satisfy((payments) => payments.every(paymentsItem =>",
+							"        Object.keys(paymentsItem).every(property => expectedPaymentTokens.has(property))",
+							"        && pm.expect(paymentsItem.amount).to.be.a(\"number\")",
+							"        && pm.expect(paymentsItem.rptId).eql(expectedWithoutAuthTokenAndTransactionId.payments[i].rptId)",
+							"        && pm.expect(paymentsItem.reason).eql(expectedWithoutAuthTokenAndTransactionId.payments[i].reason)",
+							"        && pm.expect(paymentsItem.isAllCCP).eql(expectedWithoutAuthTokenAndTransactionId.payments[i++].isAllCCP) ",
+							"    ));",
```

Comment:
Is `i` necessary?

Reply:
Not strictly necessary now, but we would be already cart compliant when we'll have cart input here.

Reply:
I choose the 🍐 option and implemented it